packages/components/src/components/hds/stepper/nav/step.hbs

```diff
@@ -0,0 +1,80 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+{{! template-lint-disable require-context-role no-invalid-role }}
+{{! template-lint-disable require-presentational-children }}
+<li class={{this.classNames}} ...attributes role={{if this.isNavInteractive "presentation"}}>
+  {{#if this.isNavInteractive}}
+    <button
+      class="hds-stepper-nav__step-content hds-stepper-nav__step-btn"
```

Comment:
We tend to avoid abbreviating words in names like "btn". I would instead spell it out or rename it.